content/en/network_monitoring/network_path/setup.md

```diff
@@ -31,8 +31,9 @@ Setting up Network Path involves configuring your Linux environment to monitor a
 
 ## Setup
 
-{{< tabs >}}
-{{% tab "Dynamic paths" %}}
+### Network Traffic Paths
```

Comment:
```suggestion
### Network traffic paths
```
Unless it's a proper noun. I see it's lowercase in the following paragraph, so this should be sentence case.